src/probnum/randprocs/markov/integrator/_ioup.py

```diff
@@ -66,7 +72,10 @@ class IntegratedOrnsteinUhlenbeckProcess(_markov_process.MarkovProcess):
     >>> ioup4 = IntegratedOrnsteinUhlenbeckProcess(driftspeed=1.,initarg=0., num_derivatives=4, wiener_process_dimension=1)
     >>> print(ioup4)
     <IntegratedOrnsteinUhlenbeckProcess with input_shape=(), output_shape=(5,), dtype=float64>
-    """
+    """  # pylint: disable=line-too-long
+    # Doctest/Example blocks in the docstring above cannot be made to comply with line
+    # length rule because adding newlines in them will cause rendered page to have
+    # unwanted newlines.
```

Comment:
As you can see below, adding line breaks in doctest lines will cause linebreaks to appear in the resulting documentation.

![image](https://user-images.githubusercontent.com/5339144/160502332-df83c3c3-be59-4630-85ee-b69bdfcea19a.png)

For example, users would see the following output when they call this class:
```
>>> ioup4 = IntegratedOrnsteinUhlenbeckProcess(driftspeed=1.,initarg=0., num_derivatives=4, wiener_process_dimension=1)
>>> print(ioup4)
<IntegratedOrnsteinUhlenbeckProcess with input_shape=(), output_shape=(5,), dtype=float64>
```

However, if I added linebreaks, the following is how it might be rendered in the final docs.


```
>>> ioup4 = IntegratedOrnsteinUhlenbeckProcess(driftspeed=1.,initarg=0., num_derivatives=4, 
wiener_process_dimension=1)
>>> print(ioup4)
<IntegratedOrnsteinUhlenbeckProcess with input_shape=(), output_shape=(5,),
dtype=float64>
```

If you are ok with the docs looking like the second block in the screenshot above, I can fix the line-too-long message for this too. I hope this made sense.

Reply:
I prefer the `pylint: disable=line-too-long` here to make sure the doctests show the expected output.

Reply:
Okay, I see. Thanks for the explanation @npate012. I agree that the linebreak is unpleasant, and in this case I have no issues with disabling line-too-long. 

One alternative could also be a linebreak with `...` such as the one in https://github.com/probabilistic-numerics/probnum/blob/0864fcfde37f2f75b46d34933552c1e44b464b80/src/probnum/diffeq/_probsolve_ivp.py#L249. 

I am okay with both solutions. What do you think?

Reply:
It is a good idea to use `...` to split up function calls to multiple lines. However, the output of the print function is also longer than 88 characters. As far as I know, `...` is only shown for your input to python. The printed output does not use `...`. If I were to break up the lines, I think one of these two lines may make sense. Would you be okay if I split up the block like below?

```python
>>> ioup4 = IntegratedOrnsteinUhlenbeckProcess(driftspeed=1.,
...                                            initarg=0.,
...                                            num_derivatives=4,
...                                            wiener_process_dimension=1)
>>> print(ioup4)
<IntegratedOrnsteinUhlenbeckProcess with input_shape=(),
                                         output_shape=(5,),
                                         dtype=float64>
```

Or if we use `...` for output too:

```python
>>> ioup4 = IntegratedOrnsteinUhlenbeckProcess(driftspeed=1.,
...                                            initarg=0.,
...                                            num_derivatives=4,
...                                            wiener_process_dimension=1)
>>> print(ioup4)
<IntegratedOrnsteinUhlenbeckProcess with input_shape=(),
...                                      output_shape=(5,),
...                                      dtype=float64>
```

I am also okay with both solutions (either using `...` or disabling line-too-long message). I will go with whichever approach you feel works best.

Thank you.

Reply:
I have kept the message ignored on this line. I have also addressed other comments.